src/sysinfo.h

```diff
@@ -4,13 +4,21 @@
 
 #include <sys/types.h>
 #include <stdbool.h>
+#include <stdint.h>
 
 typedef void* CSystem;
 typedef const void* CProcess;
 typedef const char* RString;
 typedef void* CNetworks;
 typedef void* CDisks;
 
+#ifdef WIN32
+typedef uint32_t PID;
```

Comment:
The right translation for `usize` is `unsigned int` I think.

Reply:
Maybe use `unsigned int` can be compatible with old C language. But since C99 `uint32_t` is same of `unsigned int`.

Reply:
No it's not. Most of the time it is, but nothing set in stone there. Actually, the closest type is `size_t`.

Reply:
Ok, change it.